lib/Docs/RST/RSTCopier.php

```diff
@@ -112,6 +112,14 @@ private function fixRSTSyntax(Project $project, string $content): string
         // get rid of .. include:: toc.rst
         $content = str_replace('.. include:: toc.rst', '', $content);
 
+        $content = preg_replace(
+            '/.. sectionauthor:: ([^\(]+) \(([^\)]+)\)/',
+            '.. sectionauthor:: $1 <$2>',
+            $content,
+        );
```

Comment:
Why is this necessary? Are there docs that don't follow the desired syntax already?

Reply:
Yes, some of the old docs to use `jaapio (jaap@example.com)` where it should be `jaapio <jaap@example.com>`